python/ray/data/_internal/planner/plan_udf_map_op.py

```diff
@@ -344,13 +350,20 @@ async def process_all_batches():
 
         # Yield results as they become available.
         while not future.done():
+            if not exception_queue.empty():
+                raise exception_queue.get()
             # Here, `out_batch` is a one-row output batch
             # from the async generator, corresponding to a
             # single row from the input batch.
             out_batch = output_batch_queue.get()
+            if out_batch is None:
```

Comment:
this should also be updated to `SENTINEL` i think?

Reply:
Right! I fixed! thx!